src/utils/filterPeople.ts

```diff
@@ -0,0 +1,65 @@
+import { Person } from '../types';
+
+export const filterPeople = (
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[],
+  sortField: string | null,
+  order: string | null,
+) => {
+  let tempPeople = [...people];
+
+  if (sex) {
+    tempPeople = tempPeople.filter(person => person.sex === sex);
+  }
+
+  if (centuries.length) {
+    const getCentury = (person: Person) => Math.ceil(person.born / 100);
+
+    tempPeople = tempPeople.filter(
+      person => centuries.includes(
+        getCentury(person).toString(),
+      ),
+    );
```

Comment:
Do not shorten the code entry to difficult to read, it is better to break it into logical parts.
You can also immediately return a `string` value from the `getCentury` function.
```js
const getCentury = (person: Person) => Math.ceil(person.born / 100).toString();
```
```suggestion
    tempPeople = tempPeople.filter(person => {
      const personCentury = getCentury(person);
      return centuries.includes(personCentury);
     });
```